src/components/apiUse.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+
+function FetchApi() {
+  const [quotes, setQuotes] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [errorMsg, setErrorMsg] = useState('');
+
+  useEffect(() => {
+    let isCancelled = false;
+
+    const fetchQuotes = async () => {
+      const headers = {
+        'X-Api-Key': 'aDqrgH1nQfLU2aLHs5rk4g==b0VPzAZwpMAqqzyO',
+      };
+
+      const response = await fetch(
+        'https://api.api-ninjas.com/v1/quotes?category=success',
+        { headers },
+      );
+
+      if (response.status >= 200 && response.status <= 299) {
```

Comment:
- [ ] **[OPTIONAL]** You could replace this with: 

```suggestion
      if (response.ok) {
```

According to [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok):

>The ok read-only property of the [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) interface contains a Boolean stating whether the response was successful (status in the range 200-299) or not.